### PR TITLE
Release 1.18.3 docs

### PR DESCRIPTION
**Pull request by youngnick**
Updates #4168

### CODE REVIEW

File: site/config.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ params:
   docs_versioning: true
   docs_versions:
     - main
+    - v1.18.3
```
Comment (skriss) [inline site/config.yaml:40]: Looks like the version drop-down renders things per the order in this list, so let's move this down to below v1.19.0

**Review by skriss — CHANGES_REQUESTED** [review]

File: site/content/resources/upgrading.md [hunk 2]
```diff
@@ -141,6 +141,66 @@ If your version of Contour is older than v1.18.2, please upgrade to v1.18.2 firs
     $ kubectl apply -f examples/contour/03-envoy.yaml
     ```
 
+## Upgrading Contour 1.18.2 to 1.18.3
```
Comment (sunjayBhatia) [inline site/content/resources/upgrading.md:144]: should we update the upgrade instructions after this as well? (currently 1.18.2 to 1.19.0)

File: site/data/docs/toc-mapping.yml [hunk 3]
```diff
@@ -41,3 +41,4 @@ v1.18.1: v1-18-1-toc
 v1.18.2: v1-18-2-toc
 v1.19.0: v1-19-0-toc
 v1.19.1: v1-19-1-toc
+v1.18.3: v1-18-3-toc
```
Comment (sunjayBhatia) [inline site/data/docs/toc-mapping.yml:44]: nit: move this after 1.18.2?

File: site/content/resources/upgrading.md (hunk 2 above)
Comment (youngnick) [reply site/content/resources/upgrading.md:144]: Yeah, I'll update. But this upgrade doc is increasingly terrifying to update, we may need to look at splitting it up somehow.

File: site/data/docs/toc-mapping.yml (hunk 3 above)
Comment (youngnick) [reply site/data/docs/toc-mapping.yml:44]: Thanks, that makes more sense.

**Comment by youngnick** [comment]
@skriss @sunjayBhatia updated, PTAL.

**Review by skriss — APPROVED** [review]

**Pull request merged**